client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -629,12 +626,7 @@ export const PlanComparisonGrid: React.FC< PlanComparisonGridProps > = ( {
 		if ( newVisiblePlans.length !== visibleLength ) {
 			// ensures current plan is first in the list
 			newVisiblePlans.sort( ( visiblePlan ) =>
-				[
-					currentSitePlanSlug,
-					getPlanSlugForTermVariant( currentSitePlanSlug as PlanSlug, TERM_BIENNIALLY ),
```

Comment:
I guess it could be an option to still keep this, and introduce the triannual variants too. Keeping this would force a biannual plan variant to move to the left of the comparison grid (what would happen if the plan matches the current plan), but not change the "your plan" badge or action button text. 

- So with the biannual variant included (and site on annual Business):

<img width="700" alt="Screenshot 2023-03-22 at 12 45 35 PM" src="https://user-images.githubusercontent.com/1705499/226881036-cd4df0ae-26f2-4f12-bca9-bde400ee0b90.png">

- without it (current change):

<img width="700" alt="Screenshot 2023-03-22 at 12 46 06 PM" src="https://user-images.githubusercontent.com/1705499/226881149-d0471988-11d2-493d-8621-b06c5c89d9f2.png">

cc @southp